server/src/WebSocketDataConnector.cpp

```diff
@@ -150,6 +151,7 @@ callback_isaac(
 				}
 			}
 			while (l > 1 && !lws_send_pipe_choked(wsi));
+            delete[] buf;
```

Comment:
Note: offline discussed. we will ignore the wrong white spaces for indention. It is really hard to use the right indention white space if tabs and spaces are mixed within a project. 
There will be a follow-up PR which changes the tabs to spaces.